daylight_saving.md

```diff
@@ -0,0 +1,59 @@
+NTFS Timezones and Daylight saving
+=================
+
+## Handling time-zones
+
+The `network_timezone` field in file networks.txt sets the time zone for all `stop_times` of a `trip` in the `network`. 
+
+If a feed covers multiple time zones, a specific time zone information can be specified on a `stop_point` in the `stop_timezone` field. Set this optional field in file `stops.txt` (for stops with location_type = 0). If not set, the timezone of the network is used. Ensure that trip travel times remain accurate and consistent by providing departure and arrival times based on the `network_timezone` for stops that are out of that timezone.
```

Comment:
I think the `stop_timezone` field is not used at all inside kraken/loki. 
We always use the `network_timezone` for all stop_times of a vehicle journey.

Note that allowing different timezone on each stop_time would make it harder to check if the stop_times are increasing along a trip.

Reply:
This chapter tries to describe the actual (kraken) way of dealing with TZ, and seems to be similar to what you are saying. Maybe the text is not clear enough?